packages/cli/src/workflows/workflows.services.ts

```diff
@@ -206,6 +207,17 @@ export class WorkflowsService {
 			);
 		}
 
+		// Update the workflow's hash
+		workflow.hash = uuid();
```

Comment:
If we use `@BeforeUpdate()` to set this in `WorkflowEntity`, then we ca be certain that this get updated on every update, irrespective of where it comes from in the code.

Reply:
Fair point, but we have another PR to be merged (#4200 ) and it will break if we do so. We'll be updating the `workflow_entity` on other stuff than only workflow editions, leading to more false positives. For this reason I think it's best to only update the hash when actual changes happen.